### DISCUSSION TITLE
Difference between production rates and production fluxes

### DISCUSSION THREAD

**Discussion by vgenisel** [micom-dev/micom#110] [Q&A] [answered]
Hello,
I'm new to MICOM and I've been playing with the package to simulate the effect of different diets on the production of metabolites by the gut bacteria.
My question is what is the difference between the production fluxes obtained by the function "plot_exchanges_per_sample" and the production rates obtained by "production_rates"?
I want to determine the produced microbial metabolites that could be available for host absorption, so in that case the most adequate choice should be determining the production rates?
Many thanks

**Comment by cdiener** [comment, marked as answer]
Yep, production rates are the secretion fluxes into the extracellular space. Some of those might be taken up by other microbes, but if you want to know what would be available for an organism that is (1) not part of your model right now and (2) resides in the same space as the microbes, then the production rates are exactly what you want.

**Reply by vgenisel** [reply]
Great, thanks so much for your fast reply
